frontend/components/songs/song_card_dropdown_item.jsx

```diff
@@ -32,9 +33,13 @@ const SongCardDropdownItem = ({
                 .then((playlistId) => history.push(`/playlist/${playlistId}`));
         } else if (depthLevel === 1) {
```

Comment:
What's the significance of `depthLevel` here. Is there a scenario where you might have another submenu that has a `depthLevel` of 1 as well?

Reply:
These conditionals are matching `e.target.innerText` to a specific action, but this cannot be done for all the playlists since they can all have different titles. However, for all the playlists, there would be only one action, which is add the song to that playlist if that playlist is selected. So their depthLevel can be used to select them all together here in this if statement.

You're correct, in a dropdown with multiple submenus, this wouldn't work without more selective criteria, but it can be easily modified. For this dropdown, I think it works well.